pallets/avn/src/lib.rs

```diff
@@ -541,8 +575,8 @@ impl<Balance> CollatorPayoutDustHandler<Balance> for () {
 }
 
 #[cfg(test)]
-#[path = "tests/mock.rs"]
-mod mock;
+#[path = "mock.rs"]
+pub mod mock;
```

Comment:
did you check if you need this

Reply:
It seems within the other pallets it is more common practice to not have the mock.rs file within the /tests/ subfolder